middleware/reverse/setup.go

```diff
@@ -117,8 +121,12 @@ func reverseParse(c *caddy.Controller) (nets networks, fall bool, err error) {
 				if ipnet.IP.To4() == nil {
 					regexIP = regexMatchV6
 				}
+				prefix := "^"
+				if wildcard {
+					prefix += ".*"
```

Comment:
I don't mean the tests are broken, I mean 
test.ip-127.0.3.1.example.org does not resolve to 127.0.3.1 if I set the prefix to ""
I get a SERVFAIL in the logs